kb/2025-03-25/cve-2025-1974.md

```diff
@@ -47,21 +53,49 @@ If the command returns one of the affected versions, disable the `rke2-ingress-n
     valuesContent: |-
       controller:
         admissionWebhooks:
+          port: 8444
           enabled: false
         extraArgs:
           enable-annotation-validation: true
+          default-ssl-certificate: cattle-system/tls-rancher-internal
+        config:
+          proxy-body-size: "0"
+          proxy-request-buffering: "off"
+        publishService:
+          pathOverride: kube-system/ingress-expose
   ```
 
-    Harvester automatically applies the change once the content is saved.
+   Exit the `kubectl edit` command execution to save the configuration. 
 
-RKE2 deletes the `rke2-ingress-nginx-admission` validating webhook configuration and restarts the ingress-nginx pods.
+   Harvester automatically applies the change once the content is saved.
 
-Once your Harvester cluster receives the RKE2 ingress-nginx patch, you can re-install the `rke2-ingress-nginx-admission` validating webhook configuration by removing the `HelmChartConfig` workaround.
+ :::info important
```

Comment:
This single-spaced indentation is needed to ensure the correct ordering of subsequent number bullets.